helm-chart/templates/deployment.yaml

```
@@ -38,7 +38,10 @@ spec:
         - configMapRef:
             name: {{ include "json-rpc-relay.fullname" . }}
         - secretRef:
-            name: {{ include "json-rpc-relay.fullname" . }}  
+            name: {{ include "json-rpc-relay.fullname" . }}
+        {{- with .Values.extraEnvFrom }}
```

nit: envFrom would be simpler to conform with Mirror Node actually
```suggestion
        {{- with .Values.envFrom }}
```

---

Well, to make values identical across org charts as much as possible  is a good point. Just my concern is that `envFrom` is strictly used to replace the `envFrom` of a container spec, but we have 2 items for `envFrom` in default and users will just add more items using this value. So i think `extra` would be more reasonable and meaningful.
In other charts, we can also use `envFrom` and `extraEnvFrom` according to their manifest status.

---

Makes sense. Thanks for clarifying